dashboard/app/models/course_offering.rb

```diff
@@ -385,6 +386,11 @@ def self.properties_from_file(content)
   # seed_all
   def self.seed_record(file_path)
     properties = properties_from_file(File.read(file_path))
+    if properties[:self_paced_pl_course_offering_id].present? && properties[:self_paced_pl_course_offering_key].nil?
+      warn "self_paced_pl_course_offering_key not found. Please seed again to fix."
+    end
+    properties[:self_paced_pl_course_offering_id] = CourseOffering.find_by_key(properties[:self_paced_pl_course_offering_key])&.id
```

Comment:
the warning does not look right to me. the warning should happen when the key is present, but the `CourseOffering.find_by_key` call returns nil.